tests/acceptance/poc-test.js

```diff
@@ -1,23 +0,0 @@
-import Ember from 'ember';
```

Comment:
![screen shot 2015-06-19 at 11 32 35 am](https://cloud.githubusercontent.com/assets/558005/8260216/f7d82974-1676-11e5-8f33-6ac8ae16ee81.png)
"node" and "ember" tests now run in serial